package.json

```diff
@@ -91,6 +91,7 @@
     "test-storybook:ci": "concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"yarn storybook:build && npx http-server storybook-build --port 6006 \" \"wait-on tcp:6006 && yarn test-storybook --maxWorkers=2\""
   },
   "resolutions": {
+    "@metamask/phishing-warning/ses": "^0.18.1",
```

Comment:
v1.2.2 of the phishing warning page has just been published - you can update that here to point the extension at the new page, and drop this resolution